drf_user/serializers.py

```diff
@@ -362,3 +362,19 @@ def validate(self, attrs: dict) -> dict:
             raise NotFound(_("User with the provided email does not exist."))
 
         return attrs
+
+
+class ImageSerializer(serializers.ModelSerializer):
+    """This serializer is for Image Upload API.
+
+    Params
+    profile_image: Profile Image File
+    """
+
+    profile_image = serializers.ImageField(required=True)
+
+    class Meta:
+        from .models import User
```

Comment:
@iamvivekkaushik Could you please move these imports to top level of the file and please use absolute imports.
`from drf_user.models import User`